cumulusci.yml

```diff
@@ -206,6 +206,14 @@ tasks:
             path: unpackaged/config/customer_profiles
             namespace_inject: $project_config.project__package__namespace
 
+    deploy_post_config:
+        group: "PMM: Config"
+        description: Deploys the post-install configuration to create and activate process builder processes
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/config/post
```

Comment:
Calling this package "post" I think may end up being confusing. "Post" usually means `unpackaged/post`, which is deployed automatically.

Reply:
I was initially thinking the same but thought it was within the config folder. I have deleted the `unpackaged/config/post` folder and moved `flows` into the existing `unpackaged/config/dev` folder.